src/common/options.cc

```diff
@@ -1692,6 +1692,18 @@ std::vector<Option> get_global_options() {
     .add_service("mgr")
     .set_description("Issue a health warning if there are fewer OSDs than osd_pool_default_size"),
 
+    Option("mon_warn_on_slow_ping_time", Option::TYPE_UINT, Option::LEVEL_BASIC)
+    .set_default(0)
+    .add_service("mgr")
+    .set_description("Override mon_warn_on_slow_ping_ratio with specified threshold in microseconds")
```

Comment:
Is there a reason why we cannot use seconds instead of microseconds?

Reply:
Based on typical values if we used seconds the amounts would look something like 0.000432 secs.  The ping command uses milliseconds .... 0.432 msec.  I could be convinced that we should match the ping and use milliseconds instead of microseconds.

Reply:
milliseconds makes sense to me